cterasdk/core/users.py

```diff
@@ -86,6 +90,14 @@ def add(self, name, email, first_name, last_name, password, role, company=None,
         param.role = role
         param.company = company
         param.comment = comment
+        if password_change:
+            if isinstance(password_change, bool):
```

Comment:
Can you create a common function that can be later used in other modules? Have a look in core/files/ln.py and core/files/browser.py (mklink function). The only reason I mention that is since i'm working on another module that will require that.

Perhaps we need a datetime_utils.py module and put it there. (Similar to core/union.py)

Reply:
I've created a utility module. One open question, I currently left the formating to text to the caller (in case different caller have different formats). Is that OK or not really needed